### PR TITLE
Send regression test's output with valgrind result mail

### LINKED ISSUE #23: Valgrind tests should also check and send regression.diffs
Sometimes we may miss some tests if tests fail, but we can't see any memory problems in logs.txt. Checking regression.diffs basically means we run all regressions properly. 

https://github.com/citusdata/tools/blob/develop/valgrind/run-valgrind-tests#L53

### PR DESCRIPTION
**Pull request by byucesoy**
Fixes #23

### CODE REVIEW

File: valgrind/run-valgrind-tests [hunk 1]
```diff
@@ -51,9 +51,9 @@ make check-multi-vg VALGRIND_LOG_FILE=logs.txt || true
 
 # surprisingly this hits inbox
 if [ -s logs.txt ]; then
```
Comment (metdos) [inline valgrind/run-valgrind-tests]: We should also check if regression.diffs has anything?

**Comment by byucesoy** [comment]
@metdos do you want to have another look?

**Comment by jasonmp85** [comment]
Quick question: I had an issue last month where my Heroku PG instance wasn't accepting writes, and cron fired off a bunch of mail. Did anyone get them, or did they just go locally?

**Comment by metdos** [comment]
> Quick question: I had an issue last month where my Heroku PG instance wasn't accepting writes, and cron fired off a bunch of mail. Did anyone get them, or did they just go locally?

I didn't get any emails.

**Comment by byucesoy** [comment]
> Quick question: I had an issue last month where my Heroku PG instance wasn't accepting writes, and cron fired off a bunch of mail. Did anyone get them, or did they just go locally?

I didn't get any emails too.

**Comment by metdos** [comment]
Looks good, verified that we also send regression test's output. 🚢

**Pull request merged**